src/inc/testf.h

```diff
@@ -44,6 +44,11 @@ extern unsigned int testframework_start, testframework_end;
     printf("[SUCCESS] "); \
     COLOR_RESET();
 
+#define START_TAG()            \
+    YELLOW();                  \
+    printf("[TESTF] START\n"); \
```

Comment:
The tag should [TESTF-C]. We discussed in early stages of the project that it would be the tag to represent the C side of our framework. Similar tags would be used to python tool, for instance.
Also, these tags should not have coloring, IMO. Those colors should be used for higher log_levels only.

Reply:
Thanks for these points. Fully agree with both topics. Addressed in https://github.com/bao-project/bao-tests/pull/20/commits/a8e626ca8121492c37d34dd89a9e60e148bbb78d.